<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSetLimit</class>
 <widget class="QDialog" name="frmSetLimit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置上下限</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	
	background-color: rgb(148, 148, 248);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,10">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbAbs">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>绝对值</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wdgAbs" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>上限：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAbsHi">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0.005</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>下限：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAbsLow">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0.003</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="rbPer">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>百分比</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wdgPer" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>中心值：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnPerNorminal">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QLabel" name="btnCode">
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>上限：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnPerHi">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>下限：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="btnPerLo">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>-5</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="font">
            <font>
             <family>楷体</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="title">
            <string>误差代码</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QPushButton" name="btnB">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±0.1%</string>
              </property>
              <property name="text">
               <string>&amp;B</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btnC">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±0.25%</string>
              </property>
              <property name="text">
               <string>&amp;C</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="btnD">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±0.5%</string>
              </property>
              <property name="text">
               <string>&amp;D</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="btnF">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±1%</string>
              </property>
              <property name="text">
               <string>&amp;F</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="btnG">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±2%</string>
              </property>
              <property name="text">
               <string>&amp;G</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QPushButton" name="btnJ">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±5%</string>
              </property>
              <property name="text">
               <string>&amp;J</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btnK">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±10%</string>
              </property>
              <property name="text">
               <string>&amp;K</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="btnM">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±20%</string>
              </property>
              <property name="text">
               <string>&amp;M</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="btnN">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±30%</string>
              </property>
              <property name="text">
               <string>&amp;N</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="btnW">
              <property name="minimumSize">
               <size>
                <width>41</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>楷体</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>±0.05%</string>
              </property>
              <property name="text">
               <string>&amp;W</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
